app-lifelike/src/main/java/com/example/lifelike/composable/loggedin/PhotosOfAlbum.kt

```diff
@@ -84,8 +79,43 @@ interface PhotosOfAlbum {
             val image = imageResource(R.drawable.placeholder)
 
 
+            val photoRows =  mutableListOf<List<Photo>>()
+
+            for (i in album.photos.indices step cols) {
```

Comment:
I think you can do `album.photos.chunked(cols)` here :)

Reply:
Oh fancy, didn't know that one existed